### ISSUE TITLE
💡 [REQUEST] - PyPI package?

### ISSUE THREAD

**Issue by nightlark** [LanikSJ/dfimage#252]
`pip install dfimage` would be convenient. Would you accept a PR with the changes needed to make this a pip installable package (along with a GitHub Actions workflows for publishing new versions to PyPI)?

**Comment by LanikSJ** [comment]
> Would you accept a PR with the changes needed to make this a pip installable package (along with a GitHub Actions workflows for publishing new versions to PyPI)?

Yes of course! The point of this project was to always keep `dfimage` up to date as much as my knowledge allows, which is mainly in docker world.  I'll gladly accept PRs for Python installable packages as long as you can provide the basic documentation.

I know some Python stuff if it just means adding `pip install dfimage` into the docs then I think I got the docs covered. 🙂
